makefiles/host/rootfs.mk

```diff
@@ -56,3 +56,15 @@ build/nctrl/target/release/nctrl: build/nctrl/.copy_stamp
 	FUSE_CROSS_STATIC_PATH=./thirdparty/ \
 	FUSE_CROSS_STATIC_LIB=fuse \
 	cargo build --release --target=armv7-unknown-linux-musleabihf
+
+build/root.fs/opt/openocd: build/root.fs/.base_install
+	@mkdir -p $(@D)
+	rm -rf $@
+	git clone --depth 1 https://repo.or.cz/openocd.git $@
+	touch $@/.scmversion
+
+build/root.fs/opt/openocd/.build_stamp: build/root.fs/opt/openocd
+	(cd $(@D) && ./bootstrap)
+	(cd $(@D) && ./configure --enable-sysfsgpio)
+	(cd $(@D) &&  $(MAKE) CROSS_COMPILE=$(CROSS) ARCH=$(ARCH))
```

Comment:
This is missing a `+` to pass down the jobserver.

Reply:
ah good catch